src/Nethermind/Nethermind.HealthChecks/HealthChecksPlugin.cs

```diff
@@ -140,7 +141,12 @@ public Task InitRpcModules()
 
             _nodeHealthService = new NodeHealthService(_api.SyncServer,
                 _api.MainProcessingContext!.BlockchainProcessor, _api.BlockProducerRunner!, _healthChecksConfig, _api.HealthHintService!,
-                _api.EthSyncingInfo!, _api.RpcCapabilitiesProvider, _api, drives, _initConfig.IsMining);
+                _api.EthSyncingInfo!, _engineRequestsTracker, _api.SpecProvider!.TerminalTotalDifficulty, drives, _initConfig.IsMining);
+
+            if (_mergeConfig.Enabled)
+            {
+                _engineRequestsTracker.StartAsync();
```

Comment:
Missing `await` here? (requires making the function `async`)

```suggestion
                await _engineRequestsTracker.StartAsync();
```

Reply:
This should be fire and forget

Reply:
Then let's make it explicit with `_ = _engineRequestsTracker.StartAsync(); // Fire and forget`